src/payment/create-payment-strategy-registry.ts

```diff
@@ -8,12 +8,15 @@ import { ConfigActionCreator, ConfigRequestSender } from '../config';
 import { OrderActionCreator, OrderRequestSender } from '../order';
 import { RemoteCheckoutActionCreator, RemoteCheckoutRequestSender } from '../remote-checkout';
 
-import PaymentActionCreator from './payment-action-creator';
-import PaymentMethodActionCreator from './payment-method-action-creator';
-import PaymentMethodRequestSender from './payment-method-request-sender';
-import PaymentRequestSender from './payment-request-sender';
-import PaymentStrategyActionCreator from './payment-strategy-action-creator';
-import PaymentStrategyRegistry from './payment-strategy-registry';
+import {
```

Comment:
Is there a reason we switched to this pattern here? I seem to recall previous comments against importing from `./`